src/Core/Checkout/Cart/Rule/LineItemListPriceRule.php

```diff
@@ -54,7 +54,7 @@ public function match(RuleScope $scope): bool
     public function getConstraints(): array
     {
         return [
-            'amount' => [new NotBlank(), new Type('numeric')],
```

Comment:
Can you change the request in the administration to send `amount: 0` when operator empty is used? This change would remove the check for all other operators